app/assets/javascripts/discourse/app/initializers/subscribe-user-notifications.js

```diff
@@ -108,17 +108,19 @@ export default {
         user.notification_channel_position
       );
 
-      bus.subscribe(`/user-drafts/${user.id}`, (data) => {
-        user.updateDraftProperties(data);
-      });
-
-      bus.subscribe(`/do-not-disturb/${user.get("id")}`, (data) => {
-        user.updateDoNotDisturbStatus(data.ends_at);
-      });
-
-      bus.subscribe(`/user-status/${user.id}`, (data) => {
-        user.set("status", data);
-        appEvents.trigger("user-status:changed");
+      bus.subscribe(`/user-updates/${user.id}`, (data) => {
+        switch (data.type) {
+          case "drafts":
+            user.updateDraftProperties(data.payload);
+            break;
+          case "do-not-disturb":
```

Comment:
This is kind of silly, but can we make these snake_case instead? I find kebab-case-so-annoying-to-type, also we already use snake_case in other messagebus type cases like this :) (e.g. https://github.com/discourse/discourse/blob/e245839c3c9f58e60f643b940e1cc0240735e94f/app/assets/javascripts/discourse/app/controllers/topic.js#L1694-L1697)

Reply:
Also see https://github.com/discourse/discourse-chat/blob/9c646ebd1b46c10be29ff0bb67027d6fd2c6121c/assets/javascripts/discourse/components/chat-live-pane.js#L646-L646